src/sql/migrations.rs

```diff
@@ -660,6 +660,13 @@ CREATE INDEX smtp_messageid ON imap(rfc724_mid);
             95
         ).await?;
     }
+    if dbversion < 96 {
```

Comment:
@link2xt Since I change the `acpeerstate` table here, do I also have to make a migration for your `new_acpeerstate`?